docs/gathered-data.md

```diff
@@ -797,7 +797,7 @@ API Reference:
 
 	https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/storagecluster_types.go
 
-* Location in archive: config/storage/<namespace>/<name>.json
+* Location in archive: config/storage/<namespace>/<resource>/<name>.json
```

Comment:
Will the OpenshiftStorage gatherer gather any other CRs than StorageCluster? If not, I suggest to make the path more specific and use `storagecluster(s)` instead of `<resource>`.

This might also be a good opportunity to rename the gatherer from OpenshiftStorage to StorageCluster. I don't care so much about names in the code but I do care about the section title in this document, which happens to be generated from the code.

Reply:
Same thing here. I can change it. Also, good idea to rename the Gather.